src/lib/seam/components/DeviceDetails/DeviceInfo.tsx

```diff
@@ -0,0 +1,54 @@
+import type { CommonDevice } from 'seamapi'
+
+import { BeeIcon } from 'lib/icons/Bee.js'
+import type { CommonProps } from 'lib/seam/components/common-props.js'
+import { useConnectedAccount } from 'lib/seam/connected-accounts/use-connected-account.js'
+import { DetailRow } from 'lib/ui/layout/DetailRow.js'
+import { DetailSection } from 'lib/ui/layout/DetailSection.js'
+
+interface DeviceInfoProps
+  extends Required<
+    Pick<
+      CommonProps,
+      'disableConnectedAccountInformation' | 'disableResourceIds'
+    >
+  > {
+  device: CommonDevice
+}
+
+export function DeviceInfo({
+  device,
+  disableConnectedAccountInformation,
+  disableResourceIds,
+}: DeviceInfoProps): JSX.Element | null {
+  const { connectedAccount } = useConnectedAccount(device.connected_account_id)
+  return (
+    <DetailSection label={t.deviceInfo}>
+      <DetailRow label={t.manufacturer}>
+        <div className='seam-detail-row-hstack'>
+          {device.properties.model.manufacturer_display_name}
+          {device.properties.manufacturer === 'ecobee' && <BeeIcon />}
+        </div>
+      </DetailRow>
+      {!disableConnectedAccountInformation && (
```

Comment:
This is meant to be for the whole section, not just the connect account row.

Reply:
This will hide the connected account info. The device id is not related to the connect account but may be hidden with `disableResourceIds`.

I feel the manufacturer information is fine to leave in here, but we could add a prop to hide that as well: `disableManufacturerInformation`.